{{ cookiecutter.package_name }}/tox.ini

```diff
@@ -12,6 +12,7 @@ requires =
 isolated_build = true
 indexserver =
```

Comment:
This indexserver option [is deprecated](https://tox.readthedocs.io/en/latest/config.html#conf-indexserver) so we probably shouldn't be using it here.

Reply:
Errr.... How do we set multiple env var to grab numpy-dev and astropy-dev separately from everything else?